packages/minter-contracts/ligo/src/fixed_price_sale/fixed_price_sale_market.mligo

```diff
@@ -31,15 +31,15 @@ type init_sale_param =
 }
 
 type storage = {
-    admin: pauseable_admin_storage;
+    admin: admin_storage;
     sales: (sale_param, nat) big_map;
 }
 
 type market_entry_points =
   | Sell of init_sale_param
   | Buy of sale_param
   | Cancel of sale_param
-  | Admin of pauseable_admin
+  | Admin of admin_entrypoints
 
 let transfer_nft(fa2_address, token_id, from, to_: address * token_id * address * address): operation =
```

Comment:
`transfer_nft` and `transfer_money` functions look almost identical. Consider having just one function instead of two